### PR TITLE
chore: Remove dead codes from telemetry packages. 

### PR DESCRIPTION
**Pull request by JungukCho**
**Reason for Change**:

Remove dead codes and corresponding UTs from telemetry package.

**Issue Fixed**:

**Requirements**:

- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests

**Notes**:

### CODE REVIEW

File: telemetry/telemetry.go [hunk 1]
```diff
@@ -68,36 +58,28 @@ type BridgeInfo struct {
...
+	OperationType     string
+	OperationDuration int
+	Context           string
+	SubContext        string
+	VMUptime          string
+	Timestamp         string
+	ContainerName     string
+	InfraVnetID       string
+	VnetAddressSpace  []string
+	OSDetails         OSInfo
```
Comment (tamilmani1989) [inline telemetry/telemetry.go:78]: why remove orchestratordetails but not interfgacedetails or bridgedetails

Comment (JungukCho) [reply telemetry/telemetry.go:78]: They are used in [network.go](https://github.com/Azure/azure-container-networking/blob/master/cni/network/network.go#L275)

**Review by tamilmani1989 — APPROVED** [review]
lgtm

**Pull request merged**